lib/authentic/key_manager.rb

```diff
@@ -10,14 +10,10 @@ class KeyManager
     attr_reader :store, :well_known
 
     def initialize(max_age)
-      @store = KeyStore.new(max_age)
+      @store = KeyStore.new(max_age || '10h')
```

Comment:
Instead of this `||` that requires a caller to do `KeyManager.new(nil)` to get the default, `def initialize(max_age = '10h')` would be better.

Reply:
Yea since I was providing it `nil` from opts, the default argument wasn't getting set so I went with this but I agree, bad approach, made it make more sense here bb014f58776edb3fe4acfeedeae2065c53e928ce